ci/scripts/ci_generate_documentation.sh

```diff
@@ -65,7 +65,7 @@ mv ./development/code-documentation/overview-summary.html ./development/code-doc
 grep -rl overview-summary.html development/code-documentation/ | xargs sed -i 's|overview-summary.html|index.html|g'
 
 # send development folder to project site with the documentation updated
-yarn gh-pages --dist ./development/ --dest ./development/ --add -m "docs(development): update documentation
+yarn gh-pages --dist ./development/ --dest ./development/ -m "docs(development): update documentation
```

Comment:
@Naylin15 missed symbol at the end '"'

Reply:
Hi, @ajsb85 

I create that commit with a body, specifying the folders updated

The closing symbols are at a line [71](https://git.io/fQVeS)